lib/badge-data.spec.js

```diff
@@ -45,7 +45,8 @@ describe('Badge data helpers', function() {
     forCases([
       given('gratipay', { logo: 'image/svg+xml;base64,PHN2ZyB4bWxu' }),
       given('gratipay', { logo: 'data:image/svg+xml;base64,PHN2ZyB4bWxu' }),
-      given('gratipay', { logo: 'data:image/svg xml;base64,PHN2ZyB4bWxu' }),
```

Comment:
I'm a little confused. Doesn't this function still need to do the right thing when there is a space? I could be wrong, but I imagine the query string values are already URL decoded by scoutcamp by the time they reach this function, meaning we'd receive `' '` instead of `'%20'`.

Reply:
Correct nice catch, i was on the wrong path!